### ISSUE TITLE
Signals without metadata fail to plot

### ISSUE THREAD

**Issue by tildechris** [cmu-delphi/covidcast#270]
```r
changehc = covidcast_signal(
  data_source = "changehc", 
  signal = "smoothed_adj_outpatient_covid", 
  start_day = "2020-11-04", end_day = "2020-11-04", 
  geo_type = "county")
plot(changehc)
```
Currently yields an error:

```
Error in seq.default(range[1], range[2], len = n) : 'from' must be a finite number
6. stop("'from' must be a finite number")
5. seq.default(range[1], range[2], len = n)
4. seq(range[1], range[2], len = n)
3. plot_choro(x, time_value = time_value, include = include, range = range, col = choro_col, alpha = alpha, direction = direction, dir_col = dir_col, title = title, params = choro_params)
2. plot.covidcast_signal(changehc)
1. plot(changehc)
```

Because meta-data isn't yet being returned by the API.  Manually setting the range to min/max of the value works around the error:

```
min_value <- max(changehc$value)
max_value <- min(changehc$value)

plot(changehc, range=c(max_value,min_value))
```

As per @capnrefsmmat, we should at least return a better error message in this case.

**Comment by capnrefsmmat** [comment]
The problem is this block: https://github.com/cmu-delphi/covidcast/blob/f3e3b3c12936c8d65cdebdf18c6478b41c2da5f9/R-packages/covidcast/R/plot.R#L113-L123

Since we now set `attributes(x)$metadata$geo_type` for all data frames returned from `covidcast_signal`, even those without other metadata, the check does not detect the missing mean and stdev. The check should be updated and a unit test added to verify the warning is correctly raised in this case.

**Comment by chinandrew** [comment]
This is fixed in #326
